### PR TITLE
Adding pipeline to build prototype with latest Java 19 EA build

### PR DESCRIPTION
**Pull request by jabrena**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    runs-on: ubuntu-latest
+    name: Java + Loom early access build
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v3
+      - name: setup
+        uses: oracle-actions/setup-java@v1
+        with:
+          website: jdk.java.net
+          release: Loom
```
Comment (mp911de) [inline .github/workflows/build.yml]: Can't we use directly Java 19 EAP?

Comment (jabrena) [reply .github/workflows/build.yml]: Yes, I will find the right configuration for Java 19 ea.
https://github.com/oracle-actions/setup-java

**Comment by mp911de** [comment]
Thank you for your contribution. That's merged and squashed now.

**Comment by jabrena** [comment]
Thanks you, I learn with you

**Pull request merged**